### ISSUE TITLE
Less of an issue, more of a request

### ISSUE THREAD

**Issue by NeoAscalon** [MuertoGB/SMCTool#4]
Hi,
Is it possible to add Mac Mini smc's to the program?
I have a 2014 mac mini with what happens to be a faulty smc, in fact i would like to ensure that it is indeed the smc the cause before starting a CPU swap.
I have a lot of Macbook boards but no mac mini. You see where i'm going....
The board is  820-5509-A and since the mini is a more uncommon thing to repair i thought that it would be nice to have it in the tool.
I'm a rookie when it's about programming but i can do some things, let me know if you need something, i will TRY to do some research.
Kind regards

**Comment by MuertoGB** [comment]
Currently, I've no immediate plans to add support for Mac Mini into SMC Tool, however...

If you shoot me an email at `qbug@tuta.io` and give me a few days, maybe more, I can make you a version that should achieve an 820-5509 flash. Please bare in mind I've never flashed a Mac Mini, so its a depth I've never been to; if a flash is not successful, you will have to source an exact replacement System Management Controller.

**Comment by MuertoGB** [comment]
Some research here. `Mac-35C5E08120C7EEAF` (820-5509) SMC firmware can be flashed to a MacBook, I tested with an 820-3476 board. If a donor MacBook SMC cannot get a Mac Mini to power on for flashing, you may have to program an SMC 'as-is' on a MacBook logic board before transplant onto the 5509 logic board.

**Comment by NeoAscalon** [comment]
Thank you for the quick reply.
I don't mind sordering-desoldering some SMC's , it's still fairly easy compared to a cpu,
Also i don't mind testing, it's my own machine so no problem here. Rather try than having it completely unusable.
I'll send you the mail soon.

**Comment by NeoAscalon** [comment]
Hi, quick update!
I took a look at the code, and i should be able to manage adding the board to the code and test it.
In fact i already did, but i don't have the machine with me so i will update you tomorrow if this works.
Fingers crossed!

**Comment by MuertoGB** [comment]
Okay buddy. Good work.

Will be good to know, I imagine it will flash fine. I've ordered some Mac Minis for research.

**Comment by NeoAscalon** [comment]
Hi,
It works just fine.
Thanks for according me some time.
Kind regards

**Comment by MuertoGB** [comment]
Thank you for the feedback, this is good to know.

I'm glad you got your issue resolved as well.
